lib/vrl/stdlib/src/log.rs

```diff
@@ -8,21 +8,22 @@ fn log(
     span: vrl::diagnostic::Span,
 ) -> Resolved {
     let rate_limit_secs = rate_limit_secs.try_integer()?;
+    let res = value.to_string_lossy();
```

Comment:
This requires the `json` feature of the `value` crate to be enabled. You are welcome to do so in the `Cargo.toml` file of the `stdlib` crate.